common.env

```diff
@@ -3,6 +3,8 @@ SNAPP_BASE_LIB_ADDRESS=0x9561C133DD8580860B6b7E504bC5Aa500f0f06a7
 STABLEX_CONTRACT_ADDRESS=0x9b1f7F645351AF3631a656421eD2e40f2802E6c0
 ETHEREUM_NODE_URL=http://ganache-cli:8545
 NETWORK_NAME=dev
+NETWORK_ID=42
```

Comment:
Does the 42 represent ganache/localhost as a default network?

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FJ336VCs1JC42zGRhjH%2Fgiphy.gif&token=yes" data-gitmeme-token="yes"><img src="https://media2.giphy.com/media/J336VCs1JC42zGRhjH/giphy.gif" title="Created by gitme.me with /yes"/></a>